qt/aqt/main.py

```diff
@@ -1483,7 +1494,11 @@ def onAppMsg(self, buf: str) -> Optional[QTimer]:
             if buf == "raise":
                 return None
             self.pendingImport = buf
-            return tooltip(_("Deck will be imported when a profile is opened."))
+            if is_addon:
+                msg = _("Add-on will be installed when a profile is opened.")
```

Comment:
Have you tried doing the import immediately? In the deck importing case we need an open collection, but add-on installation shouldn't require that.

Reply:
I thought I'd go the safe way and install after profile init / when mw is shown, but I just tried going the other route, and you're right, it's definitely possible to walk through the installation early on. I ran into a couple of issues, however:

- tooltips don't show up with the profile manager open – that one could be easily fixed by always using showInfo when installing from the CLI
- the prompt & progress positioning is a bit all over with the profile manager active (probably because we don't have mw raised?)
- I'm not sure about whether this could bring any issues with window activation / raising on other platforms. I'm still in the process of configuring my Windows & macOS test system for the new repo structure and build system, so I haven't been able to check myself. Do you think that could be a concern?

Reply:
Sounds like it's best to keep things simple and mimic the deck importing process for now.